### PR TITLE
Update ekuiper plugin to handle write request

### PR DESCRIPTION
**Pull request by eeff**

### CODE REVIEW

File: plugins/ekuiper/json_rw.c [hunk 1]
```diff
@@ -281,3 +283,217 @@ int json_encode_read_resp(void *json_object, void *param)
...
+        neu_json_decode_free(json_obj);
+    }
+    return ret;
+}
+
+void json_decode_write_req_free(json_write_req_t *req)
+{
+    if (NULL == req) {
+        return;
+    }
```
Comment (fengzeroz) [inline plugins/ekuiper/json_rw.c:351]: assert

File: plugins/ekuiper/plugin_ekuiper.c [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 #include "plugin_ekuiper.h"
 #include "read_write.h"
+#include <nng/nng.h>
```
Comment (fengzeroz) [inline plugins/ekuiper/plugin_ekuiper.c:25]: sort

File: plugins/ekuiper/json_rw.c (hunk 1 above)
Comment (eeff) [reply plugins/ekuiper/json_rw.c:351]: This is to keep the semantics with the C `free` function, which does nothing when freeing a NULL pointer.

File: plugins/ekuiper/plugin_ekuiper.c (hunk 2 above)
Comment (eeff) [reply plugins/ekuiper/plugin_ekuiper.c:25]: fix.

**Pull request merged**